main.py

```diff
@@ -110,7 +115,8 @@ def main(cluster_name, regions, sleep, kubeconfig, pod_namespace,
                       notifier=notifier,
                       dry_run=dry_run,
                       drainable_labels=drainable_labels,
-                      scale_label=scale_label
+                      scale_label=scale_label,
+                      instance_type_priorities=instance_type_priorities
```

Comment:
Haven't bumped into a problem there, though I did need to convert from a string->set map to a string->string map...